app/routes/metrics.py

```diff
@@ -81,15 +81,15 @@ async def get_areas_by_defined_area(
 
 @router.post("/{metric_id}/areas", response_model=List[AreasResponse])
 async def get_areas_by_polygon(
-    metric_id: Annotated[str, fastapi.Depends(metric_id_param)],
-    polygon: Polygon,
-) -> list[dict[str, float]]:
+    metric_id: str, polygon: Polygon
```

Comment:
It is better to keep the type that metric_id previously had. Or do you think there is some reason why it needs to be only string?